scipy/spatial/transform/rotation.py

```diff
@@ -281,3 +306,113 @@ def as_rotvec(self):
             return rotvec[0]
         else:
             return rotvec
+
+    @classmethod
+    def _elementary_quat_compose(cls, seq, angles, intrinsic=False):
+        # Initialize result to first axis
+        result = _make_elementary_quat(seq[0], angles[:, 0])
+
+        for idx, axis in enumerate(seq[1:], start=1):
+            if intrinsic:
+                result = _compose_quat(
+                    result,
+                    _make_elementary_quat(axis, angles[:, idx]))
+            else:
+                result = _compose_quat(
+                    _make_elementary_quat(axis, angles[:, idx]),
+                    result)
+        return result
+
+    @classmethod
+    def from_euler(cls, seq, angles, degrees=False):
+        """Initialize rotation from Euler angles.
+
+        Parameters
+        ----------
+        seq : str
```

Comment:
I think it's better to name it 'string', because there are this unicode-str issues in Python.